YouVsVirus/Assets/Scripts/LevelCollectMasks/hospitalCollider.cs

```diff
@@ -0,0 +1,28 @@
+﻿using System.Collections;
+using System.Collections.Generic;
+using UnityEngine;
+
+public class hospitalCollider : MonoBehaviour
```

Comment:
Should be Capitalized

Reply:
We do not really need this script, do we?

Reply:
nope